PackageVersionsGeneratorDefinitions.json

```diff
@@ -47,5 +47,12 @@
         "NugetPackageSearchName": "ServiceStack.Redis",
         "MinVersion": "4.0.48",
         "MaxVersion": "6.0.0"
+    },
+    {
+        "IntegrationName": "AspNetCoreMvc2",
+        "SampleProjectName": "Samples.AspNetCoreMvc2",
+        "NugetPackageSearchName": "Microsoft.AspNetCore.Mvc",
+        "MinVersion": "2.0.0",
+        "MaxVersion": "3.0.0"
```

Comment:
Is the `MaxVersion` exclusive?  `2.0.0 ≤ x < 3.0.0`

Reply:
Yes `MaxVersion` is exclusive. Would there be a better way to denote that?

Reply:
It's probably fine, as long as we are aware. My concern is that we did something different for assembly versions in the integrations, where "MaxVersion=2" (without the minor version) actually means "less than 3.0", so it can get confusing.

Reply:
Yeah you're right, and I can fix that easily. Do you mind if I address that in a separate PR?

Reply:
Sure, separate PR is ok. If it's less work to rename this to `MaxVersionExclusive` than to change the logic, that would be fine with me, too.